docs/develop/requirements-property-unified-manifest.md

```diff
@@ -0,0 +1,203 @@
+
+```json
+"extensions": [
+    ...
+    "requirements": {
+        "capabilities": [
+            {
+                "name": "MailBox",
+                "minVersion": "1.10"
+            },
+            {
+                "name": "DialogAPI",
+                "minVersion": "1.2"
+            }
+        ],
+        "formFactors": [
+            "desktop"
+        ]
+    }
+]
+```
+
+## Filter features
+
+The "requirements" properties in descendant objects of "extensions" are used to filter out; that is, block, some features of an add-in while still allowing the add-in to be installed. The implementation of this filtering is done at the source of installation, such as [AppSource](/office/dev/store/submit-to-appsource-via-partner-center) or [Microsoft 365 Admin Center](../publish/publish.md). If the version of Office doesn't support the requirements specified for the feature, then the JSON node for the feature is removed from the manifest before it is installed in the Office application. 
+
+### extensions.alternates.requirements
+
+The "extensions.alternates" property enables add-in developers to do the following:
+
+- Maintain a version of an add-in that was built on an older extensibility platform (such as COM or VSTO add-ins) or using the XML manifest, in addition to the version that uses the unified manifest.
+- Either hide or give preference to the version that uses the older technology.
+- Specify icons that are needed to make the unified manifest version of the add-in installable on Office versions that don't directly support the unified manifest. 
+
+[!INCLUDE [non-unified manifest clients note](../includes/non-unified-manifest-clients.md)]
+
+For more information, see [Manage both a unified manifest and an XML manifest version of your Office Add-in](../concepts/duplicate-legacy-metaos-add-ins.md).
+
+The "requirements" subproperty of "extensions.alternates" to selectively apply the "hide" or "prefer" subproperties only when certain requirements are met. 
+
+For example, suppose that you want to hide (from the Office UI for installing add-ins) an older version of your add-in, but only in Office versions that support the **Mailbox 1.10** requirement set. You could do that with markup similar to the following.
+
+```json
+"extensions": [
+    ...
+    {
+        ...
+        "alternates": [
+            ...
+            {
+                ...
+                "hide": {
+                    "storeOfficeAddin": {
+                        "officeAddinId": "b5a2794d-4aa5-4023-a84b-c60a3cbd33d4",
+                        "assetId": "WA999999999"
+                    }
+                },
+                "requirements": {
+                    "capabilities": [
+                        {
+                            "name": "MailBox",
```

Comment:
?? Here and other instances? Or should other places referring to this value say "MailBox" instead of "Mailbox"?

```suggestion
                            "name": "Mailbox",
```